kolibri/plugins/coach/assets/src/views/common/QuizLessonDetailsHeader.vue

```diff
@@ -72,6 +96,18 @@
       resource() {
         return this.examOrLesson === 'lesson' ? this.lesson : this.exam;
       },
+      isActive() {
+        return this.exam.active;
+      },
+      isExamDraft() {
+        return this.exam.draft;
+      },
+      isOldDataModelVersion() {
+        return this.exam.data_model_version < 3;
+      },
+      isFromOldKolibiri() {
```

Comment:
You have a small typo here - it's consistent, but let's update `isFromOldKolibiri` to `isFromOldKolibri`